angular.json

```diff
@@ -496,6 +451,40 @@
                     }
                 }
             }
+        },
+        "dotcms": {
+            "projectType": "library",
+            "root": "libs/dotcms",
+            "sourceRoot": "libs/dotcms/src",
+            "prefix": "dotcms",
+            "architect": {
+                "build": {
+                    "builder": "@angular-devkit/build-ng-packagr:build",
```

Comment:
Nx uses `@nrwl/angular:ng-packagr-lite` to build but it doesn't build (or I didn't find the option) the UMD version (that some node apps might need) so I had to change to angular's